src/processes/scoring/calc-karma-score.js

```diff
@@ -45,7 +45,13 @@ const percentileUserScore = async (targetScore, userScoreCol) => {
   return percentile;
 };
 
-const setUserKarmaScore = async (userId, combined_user_score, karma_score, userScoreCol) => {
+const setUserKarmaScore = async (
+  userId,
+  combined_user_score,
+  karma_score,
+  userScoreCol,
+  percentileScore
+) => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `setUserKarmaScore` has been modified to include a new parameter `percentileScore`. However, this parameter is not used within the function. If it's not needed, consider removing it to avoid confusion.
```diff
-  const setUserKarmaScore = async (
-    userId,
-    combined_user_score,
-    karma_score,
-    userScoreCol,
-    percentileScore
-  ) => {
+  const setUserKarmaScore = async (
+    userId,
+    combined_user_score,
+    karma_score,
+    userScoreCol
+  ) => {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->